### ISSUE TITLE
Induced Charge during dragging and jumping needs changes for JC and JW

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#358]
**While Jumping to Center**
- [x] Incorrect string for wall in the alert:
I hear "At center of play area. Negative charges in {{center of play area}} move away from Yellow Balloon."

  - Should hear: "At center of play area. Negative charges in {{wall}} move away from yellow balloon."

**Sometimes while Jumping to Center then continuing closer to wall**
- I hear, "Negative charges begin to return". I can't seem to figure out exactly when this happens. But moving right closer to wall after a Jump, should always be "Negative charges move away a little more from yellow balloon."

**Jumping to Wall**
- Just realized that jumping to wall, should always produce a more complete induced charge description (like first contact with wall). This should include string for deflection ([[STRING:Induced Charge Amount]]), string for positive charges "Positive charges do not move.", and the pattern for the Wall Object's Visual Charge. 

- [x] Specifically, it should be: "At {{wall}}. No transfer of charge. Negative charges in {{wall}} move away from {{yellow balloon}} {{a little bit}}. Positive charges do not move. Wall has many pairs of negative and positive charges."
  - This change will make jumping to wall sound the same as arriving at the wall for the first time.  

**Other Testing of Dragging and Jumping**
**Induced charge while dragging**
- Good when dragging away from wall.
- Good when dragging vertically.

**While Jumping to Near Wall**
- Sometimes a little awkward when mixing dragging and jumping, but I think good enough.

Edits made to original post (Feb 9).
Design document also updated.

**Comment by terracoda** [comment]
@jessegreenberg, actually, let's just leave the "No transfer of charge" string as it is. It's not hurint anything.

I'll update the above comment to keep "No transfer of charge."

**Comment by jessegreenberg** [comment]
> I hear, "Negative charges begin to return". I can't seem to figure out exactly when this happens.

Able to reproduce. Steps for me are
- Induce charge in wall, drag balloon close to wall.
- jump to center
- drag balloon towards wall

**Comment by jessegreenberg** [comment]
@terracoda commits in this issue have addressed comments. Closing, but please let me know if you notice anything else.
